lib/client/test.js

```diff
@@ -1,13 +1,18 @@
 import assert from 'assert/strict';
+import ClientResponse from '../client/response.js';
 import {MockClient} from './mock.js';
+import {on} from 'events';
 import StackUtils from 'stack-utils';
 
 class TestClient extends MockClient {
   constructor (options = {}) {
     super(options);
     this.res = undefined;
+    this.ws = undefined;
```

Comment:
Alphabetical order.

Reply:
Hmmmm, the existing values were not in alphabetical order, should I fix the others?